src/commands/project.js

```diff
@@ -27,25 +27,33 @@ exports.args = [{
 	}
 }];
 
-exports.handler = async ({ projectData, prs, githubAccessToken }) => {
+exports.handler = async ({ projectData, githubAccessToken }, state) => {
 	const octokit = getOctokit(githubAccessToken);
 	const { data: project } = await octokit.projects.createForOrg(projectData);
 	const { data: todoColumn } = await octokit.projects.createColumn({ project_id: project.id, name: 'To do' });
 	await octokit.projects.createColumn({ project_id: project.id, name: 'In progress' });
 	await octokit.projects.createColumn({ project_id: project.id, name: 'Done' });
 
 	await Promise.all(
-		prs.map(pr => octokit.projects.createCard({
-			column_id: todoColumn.id,
-			content_id: pr.id,
-			content_type: 'PullRequest'
-		}))
+		state.repos.map(repo => {
+			if (repo.pr) {
+				return octokit.projects.createCard({
```

Comment:
I'm getting a bit lost here. This isn't actually modifying anything in the state object is it?

If so, it'd be good to make this a `.filter().forEach()` to make it a little more clear.

Reply:
it still needs to be `map` to collect the promises for `Promise.all`. it's attaching the `project` from line 32 to the state on line 49.

Reply:
it could be `.filter().map` though

Reply:
Ah, no that's fine. I missed line 37, and was a bit thrown by the repo state to api response conversion.